src/core/event/events.types.ts

```diff
@@ -1,18 +1,27 @@
-import { PositionInfo } from "../structure/sheet.types.ts";
-import { Coordinate } from "../../utils/common.types.ts";
+import { KeyInfo } from "../structure/sheet.types.ts";
+
+export type IndexInfo = {
+  columnIndex?: number | null;
+  rowIndex?: number | null;
+};
```

Comment:
Should this be in common? Maybe `IndexPosition` instead, and fields just `column` and `row`.

Reply:
i though about keeping it just column and row, but it can get confusing what column and row is. keeping the postfix index is just easier to read when we use it. Same goes for the keyInfo

Reply:
what do you mean by common? you mean sheet.types.ts?